pkg/envoy/lds/response.go

```diff
@@ -31,7 +31,7 @@ func NewResponse(meshCatalog catalog.MeshCataloger, proxy *envoy.Proxy, _ *xds_d
 		statsHeaders = proxy.StatsHeaders()
 	}
 
-	svcList, err := proxyRegistry.ListProxyServices(proxy)
+	svcList, err := meshCatalog.GetServicesForProxy(proxy)
```

Comment:
nit: by convention, our code seems to use `Get` for singular queries and `List` for plural queries. Is there a reason for the switch here?

Reply:
will make the change back to List